packages/hawtio/scripts/generate-camel-svg-index.js

```diff
@@ -1,114 +1,113 @@
 /* jshint node: true */
-import fs from 'fs'
-import path from 'path'
-import { fileURLToPath } from 'url';
-
-const ROOT = '..'
-const __filename = fileURLToPath(import.meta.url);
-const __dirname = path.dirname(__filename);
-const svgdir = path.resolve(__dirname, ROOT, 'src', 'plugins', 'camel', 'icons', 'svg')
-const index = path.resolve(svgdir, 'index.ts')
-
-function capitalize(value) {
-  return value.charAt(0).toUpperCase() + value.slice(1)
-}
-
-function safeSvgName(name) {
-  let iname = name.replace('.svg', '') // Remove svg file extension
-  iname = iname.replace('24', '') // Remove 24 suffix
-  iname = iname.replace('-icon', '') // Remove any -icon suffix
-  iname = iname.replace('icon', '') // Remove remaining icon suffix
-  iname = iname.replace(/-([a-z])/g, g => {
-    return g[1].toUpperCase()
-  })
-  iname = iname.charAt(0).toLowerCase() + iname.slice(1) // Lower-case the first letter
-  return iname
-}
-
-async function generateContents(svgPath, prefix) {
-  const impContent = []
-  const expContent = []
-  const expNames = []
-  const names = []
-
-  const dir = await fs.promises.opendir(svgPath)
-  for await (const ent of dir) {
-    if (ent.isDirectory()) {
-      const contents = await generateContents(path.resolve(svgPath, ent.name), ent.name)
-      impContent.push(...contents[0])
-      expContent.push(...contents[1])
-      expNames.push(...contents[2])
-    }
+const fs = import('fs')
+const path = import('path')
 
-    if (!ent.name.endsWith('.svg')) {
-      continue
-    }
+Promise.all([fs, path]).then(([fs, path]) => {
```

Comment:
It seems like a lot of changes but I only moved the whole code inside the Promise resolve